assets/css/components/_post-article.scss

```diff
@@ -0,0 +1,43 @@
+.post-article {
+  .author {
+    color: $grey-light;
+    font-style: italic;
+  }
+
+  .date {
+    margin-bottom: rem(32);
+  }
+
+  .api-content {
+    border-bottom: 2px solid $green-dark;
+    margin-bottom: rem(24);
+    padding-bottom: rem(160);
+  }
+
+  .tags-info {
+    h3 {
+      color: $green-dark;
+      margin: rem(8);
```

Comment:
I'd remove the left and right margin as this creates a slight indent which looks odd.